File: SDKs/amazonaws.com/transfer/2018-11-05/python/src/sdk/models/shared/describeserverresponse.py
from dataclasses import dataclass, field
from dataclasses_json import dataclass_json
from . import describedserver


@dataclass_json
@dataclass
class DescribeServerResponse:
    server: describedserver.DescribedServer = field(default=None, metadata={'dataclasses_json': { 'field_name': 'Server' }})
    

File: SDKs/amazonaws.com/transfer/2018-11-05/typescript/src/sdk/models/shared/describeserverresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { DescribedServer } from "./describedserver";



export class DescribeServerResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=Server" })
  server: DescribedServer;
}
